Formula/httpie.rb

```diff
@@ -1,45 +1,44 @@
+# brew install https://gist.githubusercontent.com/jkbrzt/e59a0cde6ae6f59f2182/raw/httpie.rb
+
 class Httpie < Formula
   desc "User-friendly cURL replacement (command-line HTTP client)"
   homepage "http://httpie.org/"
-  url "https://github.com/jkbrzt/httpie/archive/0.9.3.tar.gz"
-  sha256 "2a0c7cf6a6914620eebc2d7700e8e7a57aabde62bd62cd7fa68f8b216c0b2340"
+  url "https://pypi.python.org/packages/70/91/47c81b650d3cf33e7e9e639c315903947466a1993053fe1ff019f555ca97/httpie-0.9.4.tar.gz"
+  sha256 "0fc288a85d6c018c64bbc86dfcc9c7fad063e79816840dfa91e8d6c43654761e"
 
   head "https://github.com/jkbrzt/httpie.git"
 
-  bottle do
```

Comment:
Please don't remove `bottle do` blocks on update. CI knows to ignore existing bottles.